web/lib/src/helpers/extensions.dart

```diff
@@ -70,6 +70,7 @@ extension CanvasRenderingContext2DGlue on CanvasRenderingContext2D {
 }
```

Comment:
Thoughts on deprecating some of the other "glue" extensions?

Reply:
`drawImageScaled` - sure.

The other canvas bits seem...fine?

Reply:
Fair - if they're doing nontrivial work, it makes sense to keep them.